server/controllers/filterLogController.js

```diff
@@ -0,0 +1,52 @@
+const Logs = require('../models/logs');
+
+const filterLogController = {};
+
+filterLogController.filter = async (req, res, next) => {
+  try {
+    const { level, startSearch, keyword } = req.body;
+
+    if (!level && !startSearch && !keyword) {
+      const dataError = {
+        log: 'filterLogController middleware: no target for filtering',
```

Comment:
This should be controlled on the FE and also where the filters are empy here we should return all logs by default.